CHANGELOG.md

```
@@ -2,6 +2,22 @@
 
 This contains only the most important and/or user-facing changes; for a full changelog, see the commit history.
 
+## [0.13.0](https://github.com/ably/ably-chat-js/tree/0.13.0) (2025-09-17)
+
+### New Features
+
+- **Client Message Reactions Fetching**: Added method to fetch a client's reactions ("my reactions") in preparation for clipping at large scale. [#626](https://github.com/ably/ably-chat-js/pull/626)
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)